sklearn/linear_model/_least_angle.py

```diff
@@ -31,7 +31,7 @@
 
 @_deprecate_positional_args
 def lars_path(X, y, Xy=None, *, Gram=None, max_iter=500, alpha_min=0,
-              method='lar', copy_X=True, eps=np.finfo(float).eps,
+              method='lars', copy_X=True, eps=np.finfo(float).eps,
```

Comment:
We can not change this without a deprecation cycle, because this change is not backward compatible.

For this specific case, I do not it is worth a deprecation cycle to make this name change.